scripts/docker/run

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# https://www.linuxjournal.com/content/validating-ip-address-bash-script
+function valid_ip()
+{
+  local  ip=$1
+  local  stat=1
+
+  if [[ $ip =~ ^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$ ]]; then
+    OIFS=$IFS
+    IFS='.'
+    ip=($ip)
+    IFS=$OIFS
+    [[ ${ip[0]} -le 255 && ${ip[1]} -le 255 \
+      && ${ip[2]} -le 255 && ${ip[3]} -le 255 ]]
+    stat=$?
+  fi
+  return $stat
+}
+
+PUB_IP=$(wget -qO- https://diagnostic.opendns.com/myip)
```

Comment:
Does it return ipv4 address only?  Check my PR. https://github.com/harmony-one/harmony/pull/715